app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -664,9 +671,9 @@ void OpenXRInputSource::EmulateControllerFromHand(device::RenderMode renderMode,
     }
     delegate.SetPinchFactor(mIndex, pinchFactor);
     delegate.SetButtonState(mIndex, ControllerDelegate::BUTTON_TRIGGER,
-                            device::kImmersiveButtonTrigger, indexPinching && !leftPalmFacesHead,
-                            indexPinching && !leftPalmFacesHead, 1.0);
-    if (leftPalmFacesHead) {
+                            device::kImmersiveButtonTrigger, indexPinching && !palmFacesHead,
+                            indexPinching && !palmFacesHead, 1.0);
+    if (palmFacesHead) {
```

Comment:
Also this is probably wrong too, I don't think we should set those values when the right hand is facing the head.

Reply:
Oops, right. That's an existing bug I introduced when trying to fix the single hand locking out the trigger button.

Great if you could fix it as part of this PR :sweat_smile:

Reply:
> Oops, right. That's an existing bug I introduced when trying to fix the single hand locking out the trigger button.
> 
> Great if you could fix it as part of this PR sweat_smile

Please ignore this comment. There is a bug indeed but is not related with this PR.

Reply:
I meant that there is indeed an issue, any hand is not able to trigger a click without having a valid aim state. But it is not related with this PR, so please ignore my comments above. Sorry for the noise.